src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/UpdateVCFSequenceDictionaryIntegrationTest.java

```diff
@@ -36,9 +36,9 @@
                 // can't handle CRAM - see https://github.com/samtools/htsjdk/issues/731
                 new Object[]{ new File(testDir, "variantsNoDict.vcf"), new File(testDir, "exampleBAM.bam"), null, null, false, false},
                 // already has a dictionary - but force a replace
-                new Object[]{ new File(testDir, "variantsWithDict.vcf"), new File(testDir, "exampleFASTA.dict"), null, null, true, false},
+                new Object[]{ new File(testDir, "variantsWithDict.vcf"), new File(testDir, "exampleFASTA.dict"), null, null, true, true},
```

Comment:
@cmnbroad the changes I made here are catching a contig length mismatch in these tests.  Given that the subsequent test, where the input VCF has no lengths defined in the dictionary, requires turning off validation, I thought that might be acceptable here, but I wanted to check with you as test author.

Reply:
@ldgauthier Seems fine. It looks like this same test case is duplicated on the next line though and can be removed while you're in there since its redundant.

Reply:
@cmnbroad I hate to delete a test.  Should I add one where we overwrite a valid dictionary with a (matching) valid dictionary?  That seems like what that original test was intending to do, although that's not particularly interesting behavior.

Reply:
@ldgauthier Sure - it seems easy enough to add that. I was mostly just saying that we don't need to have the exact same test case appear in the list twice, which is what we had before, and still have now.